internal/cmd/root.go

```diff
@@ -24,8 +25,12 @@ func NewRootCmd() *cobra.Command {
 		Short:        "A cli for executing SQL statements on a libSQL or SQLite database",
 		Args:         cobra.MatchAll(cobra.ExactArgs(1), cobra.OnlyValidArgs),
 		RunE: func(cmd *cobra.Command, args []string) error {
+			dbUri := args[0]
+			if rootArgs.authToken != "" {
+				dbUri = fmt.Sprintf("%s?authToken=%s", dbUri, rootArgs.authToken)
+			}
```

Comment:
dbUri can have another query parameter. So the safer and simpler way is to do a `url.Parse` and do operations at `url.URL` struct